corehq/apps/registration/templates/registration/partials/register_new_user/submit_info.html

```diff
@@ -10,12 +10,12 @@ <h2>{% trans "Creating Account..." %}</h2>
       Please wait a moment&mdash;almost there!
     {% endblocktrans %}
   </p>
+  <p class="timeout-message" data-bind="visible: showSecondTimeout">
```

Comment:
Are these going to show in the right order now that the second timeout is above the first timeout?

Reply:
the order doesn't really matter as these messages are fake anyway?

Reply:
I mean, is the user first only going to see "Created sample applications..." and then "Created new user..." will appear above it?